packages/core/src/lib/connection_manager.ts

```diff
@@ -309,9 +302,12 @@ export class ConnectionManager {
    * Fetches the tag names for a given peer
    */
   private async getTagNamesForPeer(peerId: PeerId): Promise<string[]> {
-    const tags = (await this.libp2pComponents.peerStore.getTags(peerId)).map(
-      (tag) => tag.name
-    );
-    return tags;
+    try {
+      const peer = await this.libp2p.peerStore.get(peerId);
```

Comment:
context: Now all methods on `PersistentPeerStore` that `libp2p` uses are throwing so it  is up to us how we handle that.